### ISSUE TITLE
Do not render error when calling templater script inside the input

### ISSUE THREAD

**Issue by MarceloCajueiro** [mProjectsCode/obsidian-meta-bind-plugin#47]
-   [X] The Plugin is up to date
-   [X] Obsidian is up to date

**Describe the bug**
When calling an templater script, it show the following error:

![image](https://user-images.githubusercontent.com/26076/209557232-bf5045cd-9c82-42f3-95e6-4f2e6959f8c9.png)

**To Reproduce**
Call templater script inside the input. For example:

```
INPUT[multi_select(
  <% tp.user.options("journals/_props/domestic_tasks.md") %>
):Domestic_tasks]
```

**Expected behavior**

Print the code instead of the input when has a script inside.

**Comment by MarceloCajueiro** [comment]
Thinking better, can be a feature and not a bug fix.

**Comment by mProjectsCode** [comment]
I guess this can be closed then in favor of #48
